changelog/1477.trivial.rst

```diff
@@ -0,0 +1,6 @@
+Separated null_point_find into two functions named null_point_find
+and uniform_null_point_find in order to provide more clarity to the user.
+null_point_find will find the null points of a vector space whose values
+are manually entered.
+uniform_null_point_find will find the  null points of a uniform vector
+space whose values are generated by a function inputed by the user.
```

Comment:
```suggestion
Separated ``plasmapy.analysis.nullpoint.null_point_find`` into
two functions named
`~plasmapy.analysis.nullpoint.null_point_find` and
`plasmapy.analysis.nullpoint.uniform_null_point_find` in order to
provide more clarity to the
user. `~plasmapy.analysis.nullpoint.null_point_find` will find
the null points of a vector space whose values are manually
entered.  `plasmapy.analysis.nullpoint.uniform_null_point_find`
will find the null points of a uniform vector space whose values
are generated by a function inputted by the user.
```
Adding in some [reStructuredText links](https://docs.plasmapy.org/en/latest/development/doc_guide.html#restructuredtext-examples) to the functions.